pyhaproxy/test.py

```diff
@@ -153,47 +153,47 @@ def teardown(self):
         pass
 
     def test_parse_global_section(self):
-        print self.configration
```

Comment:
@alisson276 thanks for the update. Maybe we need to add an extra `from __future__ import print_function` at the top line of the file, which guarantees us using print as a function.

Reply:
@imjoey , it's done!